### PR TITLE
Importing AudioBuffer from PositionalAudio.

### PR DESCRIPTION
**Pull request by JasonTartaglia**
Fixes an issue in TypeScript where AudioBuffer was getting defined from lib.dom.d.ts.

### CODE REVIEW

File: src/audio/Audio.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Object3D } from './../core/Object3D';
 import { AudioListener } from './AudioListener';
 import { AudioContext } from './AudioContext';
+import { AudioBuffer } from "./PositionalAudio";
```
Comment (Mugen87) [inline src/audio/Audio.d.ts]: How about moving the `AudioBuffer` to `Audio.d.ts`? It seems it is now used in `PositionalAudio`.

Comment (JasonTartaglia) [reply src/audio/Audio.d.ts]: Sure. I wasn't sure why it was there but didn't want to make any assumptions. I'll fix it up now.

Comment (JasonTartaglia) [reply src/audio/Audio.d.ts]: @Mugen87 Okay. Should be good to go now. I moved the definition for AudioBuffer into the Audio.d.ts file and removed the import to PositionalAudio.

Comment (JasonTartaglia) [reply src/audio/Audio.d.ts]: Whoops... my Prettier kicked in - I'll switch back to match existing style.

Comment (JasonTartaglia) [reply src/audio/Audio.d.ts]: Okay... used this: http://zz85.github.io/mrdoobapproves/
Hope that makes this style compliant now.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**